app/assets/stylesheets/application.scss

```diff
@@ -1,3 +1,4 @@
+// Tailwind
 @import "tailwindcss/base";
 @import "tailwindcss/components";
-@import "tailwindcss/utilities";
+@import "tailwindcss/utilities";
```

Comment:
The location should be ` app/javascript/stylesheets/application.scss`

Reply:
Restored our `application.scss` back to `JS/Stylesheets`